app/api/chat/route.ts

```diff
@@ -54,23 +54,33 @@ export const POST = withWidgetAuth(async ({ request }, { session, mailbox }) =>
   const conversation = await getConversation(conversationSlug, session);
 
   const userEmail = session.isAnonymous ? null : session.email || null;
-  const screenshotData = message.experimental_attachments?.[0]?.url;
-
-  if (
-    (message.experimental_attachments ?? []).length > 1 ||
-    (screenshotData && !screenshotData.startsWith("data:image/png;base64,"))
-  ) {
-    return corsResponse(
-      { error: "Only a single PNG image attachment sent via data URL is supported" },
-      { status: 400 },
-    );
+  const attachments = message.experimental_attachments ?? [];
+
+  // Validate attachments
+  for (const attachment of attachments) {
+    if (!attachment.url.startsWith("data:image/") || !attachment.url.includes(",")) {
```

Comment:
We can be a bit stricter about this I think, e.g.

```suggestion
    if (!/^data:image\/(png|jpeg|gif|webp);base64,/.test(attachment.url)) {
```

(not sure what mimetypes exactly we should support, but you get the idea)